Hatnya/Screens/CreateStudy/UIComponent/SelectCycleDaysView.swift

```diff
@@ -40,4 +40,48 @@ final class SelectCycleDaysView: UIView {
             dayButtonStackView.bottomAnchor.constraint(equalTo: self.bottomAnchor)
         ])
     }
+    
+    func returnSelectedCycle() -> Int {
+        switch cycleSegmentedControlView.selectedCycle {
+        case "1주":
+            return 1
+        case "2주":
+            return 2
+        case "3주":
+            return 3
+        case "4주":
+            return 4
+        default:
+            return 0
+        }
+    }
```

Comment:
```suggestion
    func getSelectedCycle() -> Int {
        switch cycleSegmentedControlView.selectedCycle {
        case .one:
            return 1
        case .two:
            return 2
        case .three:
            return 3
        case .four:
            return 4
        }
    }
```

위의 cycleSegmentedControlView.selectedCycle를 rawValue가 아닌 enum으로 바꾸고
switch문에 enum을 이용하는게 더 적절해 보여요 !
함수명도 살짝 고치면 좋을 것 같아요 !

Reply:
[ef8f6ae](https://github.com/DeveloperAcademy-POSTECH/MC3-Team1-Hatnya/pull/54/commits/ef8f6ae6a4895b85ee966753236ff865c3a572d3)
enum 적용
[fcbfd23](https://github.com/DeveloperAcademy-POSTECH/MC3-Team1-Hatnya/pull/54/commits/fcbfd23fab95bfbba40b21a29c91e737046e2420)
함수명 변경!!

Reply:
멋쟁이 👍